### ISSUE TITLE
Editing tree of non-selectable type

### ISSUE THREAD

**Issue by quite** [fruktkartan/fruktkartan#49]
Editing a tree that is of a type that is not selectable in the UI could be confusing. Tree type is "_Trädsort_". It does work but... Hm

**Comment by rotsee** [comment]
Right, that's a bug. The custom tree type would need to be added to the select menu in such cases.
Or we could just purge those trees from the database, or merge them with allowed tree types when possible. What do you think?

**Comment by quite** [comment]
Ah yes, one could just add it temporarily there.

No6f they need to remain, in general. My case here was körsbärskornell, the
cornelian cherry (which I had to set manually in the db :)

**Comment by quite** [comment]
How do we do this? It's the kind of thing I find hard to grasp with vue. When the `v-select` is "mounted", it should probably check if `tree.type` is present in `items` (`insertableTrees`). If it is not, it should be added there.

**Comment by rotsee** [comment]
I can have a look this weekend, but I'm guessing the easiest way is to make the list of items a [computed property](https://vuejs.org/v2/guide/computed.html)
